.github/workflows/test-reports.yml

```diff
@@ -0,0 +1,81 @@
+name: Test Reports
+
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Just so I understand: normal test suite will run on all pushes to PRs. The coverage suite will only run when we merge (we're never pushing directly to develop that I'm aware of)?

Reply:
The PR workflow will run on all PRs.  This version of the PR workflow will run on push to develop and main which is effectively the merge to those respective branches.